src/main/java/com/flightstats/hub/metrics/InfluxdbReporterProvider.java

```diff
@@ -3,53 +3,65 @@
+    private final MetricsProperties metricsProperties;
     private final MetricRegistry metricRegistry;
+    private final HubVersion hubVersion;
 
     @Inject
-    public InfluxdbReporterProvider(MetricsConfig metricsConfig, MetricRegistry metricRegistry) {
-        this.metricsConfig = metricsConfig;
+    public InfluxdbReporterProvider(TickMetricsProperties tickMetricsProperties,
+                                    MetricsProperties metricsProperties,
+                                    MetricRegistry metricRegistry,
+                                    HubVersion hubVersion) {
+        this.tickMetricsProperties = tickMetricsProperties;
+        this.metricsProperties = metricsProperties;
         this.metricRegistry = metricRegistry;
+        this.hubVersion = hubVersion;
     }
 
     private UdpInfluxdbProtocol udpProtocol() {
-        return new UdpInfluxdbProtocol(metricsConfig.getInfluxdbHost(), metricsConfig.getInfluxdbPort());
+        return new UdpInfluxdbProtocol(tickMetricsProperties.getInfluxDbHost(), tickMetricsProperties.getInfluxDbPort());
     }
 
     private HttpInfluxdbProtocol httpProtocol() {
         return new HttpInfluxdbProtocol(
-                metricsConfig.getInfluxdbHost(),
-                metricsConfig.getInfluxdbPort(),
-                metricsConfig.getInfluxdbUser(),
-                metricsConfig.getInfluxdbPass(),
-                metricsConfig.getInfluxdbDatabaseName());
+                tickMetricsProperties.getInfluxDbHost(),
+                tickMetricsProperties.getInfluxDbPort(),
+                tickMetricsProperties.getInfluxDbUser(),
+                tickMetricsProperties.getInfluxDbPassword(),
+                tickMetricsProperties.getInfluxDbName());
     }
 
     @Override
     public ScheduledReporter get() {
-        String protocolId = metricsConfig.getInfluxdbProtocol();
+        String protocolId = tickMetricsProperties.getInfluxDbProtocol();
         InfluxdbProtocol protocol = protocolId.contains("http") ? httpProtocol() : udpProtocol();
         return InfluxdbReporter.forRegistry(metricRegistry)
-            .protocol(protocol)
-            .convertRatesTo(TimeUnit.SECONDS)
-            .convertDurationsTo(TimeUnit.MILLISECONDS)
-            .filter(MetricFilter.ALL)
-            .skipIdleMetrics(false)
-            .tag("role", metricsConfig.getRole())
-            .tag("team", metricsConfig.getTeam())
-            .tag("env", metricsConfig.getEnv())
-            .tag("cluster", metricsConfig.getClusterTag())
-            .tag("host", metricsConfig.getHostTag())
-            .tag("version", metricsConfig.getAppVersion())
-            .build();
+                .protocol(protocol)
+                .convertRatesTo(TimeUnit.SECONDS)
+                .convertDurationsTo(TimeUnit.MILLISECONDS)
+                .filter(MetricFilter.ALL)
+                .skipIdleMetrics(false)
+                .tag("role", metricsProperties.getTagsRole())
```

Comment:
I think that it might be a little bit more consistent to change these two names to getRoleTag, getTeamTag OR change getClusterTag to match these, I prefer the former personally